add some more of the limitations documented elsewhere to this page as well (#123425)
Azure/azure-content: articles/azure-monitor/visualize/workbooks-bring-your-own-storage.md

- When you save to custom storage, you can't pin individual parts of the workbook to a dashboard because the individual pins would contain protected information in the dashboard itself. When you use custom storage, you can only pin links to the workbook itself to dashboards.
- After a workbook has been saved to custom storage, it will always be saved to custom storage, and this feature can't be turned off. To save elsewhere, you can use **Save As** and elect to not save the copy to custom storage.

> - Workbooks saved to custom storage cannot be recovered by the support team. Users may be able to recover the workbook content if soft-delete or blob versioning are enabled on the underlying storage account. See [recovering a deleted workbook](workbooks-manage.md#recover-a-deleted-workbook) 

Reviewer: ```suggestion
- Workbooks saved to custom storage can't be recovered by the support team. Users might be able to recover the workbook content if soft-delete or blob versioning is enabled on the underlying storage account. See [recovering a deleted workbook](workbooks-manage.md#recover-a-deleted-workbook).
```

> - Workbooks saved to custom storage do not support versioning, only the most recent version is stored. Other versions may be available in storage if blob versioning is enabled on the underlying storage account.  See [manage workbook versions](workbooks-manage.md#manage-workbook-versions).

Reviewer: ```suggestion
- Workbooks saved to custom storage don't support versioning. Only the most recent version is stored. Other versions might be available in storage if blob versioning is enabled on the underlying storage account.  See [manage workbook versions](workbooks-manage.md#manage-workbook-versions).
```
